.changeset/quick-cats-fix.md

```diff
@@ -0,0 +1,6 @@
+---
+"remix": patch
+"@remix-run/testing": patch
+---
+
+remove internal installGlobals function now that @remix-run/web-form-data now includes support for passing a HTMLFormElement now
```

Comment:
Too many `now`'s 🙈

```suggestion
remove internal `installGlobals` function now that `@remix-run/web-form-data` includes support for passing a `HTMLFormElement`
```

Reply:
copy paste without any review lol, thanks for the fix